torntools/scripts/content/global/ttGlobal.js

```diff
@@ -1022,14 +1022,15 @@ function showNpcLoot() {
 		}
 		let npcLootLevel, npcNextLevelIn;
 		if (npcInHosp) {
+			let hospOutIn = npcData.hospout * 1000 - Date.now();
 			npcLootLevel = doc.new({ type: "span", class: "loot", text: "0" });
-			npcNextLevelIn = doc.new({ type: "span", text: timeUntil(npcData.hospout * 1000 - Date.now()) });
+			npcNextLevelIn = doc.new({ type: "span", text: timeUntil(hospOutIn), attributes: {seconds: hospOutIn} });
 		} else {
 			for (let lootLevel in npcData.timings) {
 				let nextLvlTime = npcData.timings[lootLevel].ts * 1000 - Date.now();
 				if (nextLvlTime > 0) {
 					npcLootLevel = doc.new({ type: "span", class: "loot", text: lootLevel - 1 });
-					npcNextLevelIn = doc.new({ type: "span", text: timeUntil(nextLvlTime) });
+					npcNextLevelIn = doc.new({ type: "span", text: timeUntil(nextLvlTime), attributes: {seconds: nextLvlTime}});
```

Comment:
I'm pretty sure `nextLvlTime` is in milliseconds.

Reply:
Hence I subtracted 1000 in the interval.
It is only the naming.

Reply:
Yeah, but you are setting the value `seconds` to milliseconds. I doubt it would be correct, although I can't test due to the downtime atm

Reply:
Changed it.

Reply:
I tested it and it was working so I didn't change it.